inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongGroupApi.java

```diff
@@ -68,4 +69,6 @@ public interface InlongGroupApi {
     @DELETE("group/delete/{id}")
     Call<Response<Boolean>> deleteGroup(@Path("id") String id);
 
+    @POST("group/reset")
```

Comment:
Please add the ut of this, it is very simple, an example can refer to 

org.apache.inlong.manager.client.api.inner.InnerInlongManagerClientTest

Reply:
ok